net/http_ping/files/makefile-support-macports-openssl3.diff

```diff
@@ -0,0 +1,17 @@
+--- Makefile	2014-08-11 19:13:31.000000000 +0000
++++ Makefile	2024-11-24 19:47:54.834320939 +0000
+@@ -9,10 +9,10 @@
+ # http://www.openssl.org/  Make sure the SSL_TREE definition points to the
+ # tree with your OpenSSL installation - depending on how you installed it,
+ # it may be in /usr/local instead of /usr/local/ssl.
+-#SSL_TREE =	/usr/local/ssl
+-#SSL_DEFS =	-DUSE_SSL
+-#SSL_INC =	-I$(SSL_TREE)/include
+-#SSL_LIBS =	-L$(SSL_TREE)/lib -lssl -lcrypto
++SSL_TREE =	/opt/local/libexec/openssl3
```

Comment:
It's not idea to hardcode `/opt/local`. It's the default prefix for MacPorts, but won't always be the case for every MacPorts installation.

The `${prefix}` variable in the Portfile will always point to the correct prefix for the current MacPorts installation.

I think that that might be available inside `make` as `$(PREFIX)`.  It might be possible to set this as:

```suggestion
SSL_TREE =	$(PREFIX)/libexec/openssl3
```

I'm not sure if that will work, but it's worth a shot.

Reply:
This is a small oversight. I'll get to fixing it later.